lib/data/network/mcnd_api.dart

```diff
@@ -12,27 +12,27 @@ class McndApi {
   McndApi(this._dio, this._mapper);
 
   Future<List<ApiPrayerTime>> getPrayerTime(PrayerTimeFilter filter) async {
-    final response = await _dio.get(
-      "prayertime",
-      queryParameters: {
-        "filter": filter.getApiStringValue(),
+    final Response<dynamic> response = await _dio.get<dynamic>(
```

Comment:
if the rule enforcing this style I thin we'd better change it or ignore it as I don't think this is a good practice!

Reply:
Why, the type should be clear, so it can be inferred.

We can make it like this:
```dart
final Response response = await _dio.get<dynamic>(....);
```
or
```dart
final response = await _dio.get<dynamic>(....);
```

Reply:
I meant for <dynamic>

Reply:
?